src/Uno.UWP/Globalization/Calendar.cs

```diff
@@ -351,6 +351,10 @@ public void SetToMin()
 		public void SetToMax()
 		{
 			var calendarMaxSupportedDateTime = _calendar.MaxSupportedDateTime;
+			if (calendarMaxSupportedDateTime == DateTime.MaxValue)
+			{
+				calendarMaxSupportedDateTime = calendarMaxSupportedDateTime.AddYears(-1);
```

Comment:
Runtime test? :D

Reply:
@kazo0 CI machines are UTC, so the crash will hardly be reproducible there :'(